AudioComments/AudioComments/AudioCommentView/AudioCommentView.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  AudioCommentView.swift
+//  AudioComments
+//
+//  Created by Shawn Gee on 5/5/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+import UIKit
+
+protocol AudioCommentViewDelegate: AnyObject {
+    func startRecording(for audioCommentView: AudioCommentView)
+    func stopRecording(for audioCommentView: AudioCommentView)
+    func startPlayback(for audioCommentView: AudioCommentView)
+    func pausePlayback(for audioCommentView: AudioCommentView)
+    func scrubPlayback(to location: Float, for audioCommentView: AudioCommentView)
+    
+    func sendAudioComment(for audioCommentView: AudioCommentView)
+    func sendTextualComment(with text: String, for audioCommentView: AudioCommentView)
+}
```

Comment:
So it looks like you're planning to put the actual recorder object on the tableVC, and control it through this interface? Interesting! I would have put the recorder on this view itself, but I can see some benefits to doing it this way.